packages/react/src/components/SuiteHeader/SuiteHeaderProfile/__snapshots__/SuiteHeaderProfile.story.storyshot

```diff
@@ -228,7 +228,7 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT/Suite
         >
           <button
             className="iot--btn bx--btn bx--btn--sm bx--btn--secondary"
-            data-testid="suite-header-profile--profile"
```

Comment:
same thing here, should we add this testid?

Reply:
Done